code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -318,6 +318,13 @@
 		return
 	..()
 
+/obj/item/pizzabox/AltClick(mob/user)
+	..()
+	if(length(boxes) || !Adjacent(user))
+		return
+	open != open 
```

Comment:
```suggestion
	open = !open 
```
not sure why your previous commit didn't apply correctly, but whatever. One last change.